docs/source/reference/evaql/create_job.rst

```diff
@@ -0,0 +1,41 @@
+CREATE JOB
+===============
+
+.. _create-job:
+
+The CREATE JOB statement allows us to create recurring or one-time jobs in EvaDB. These jobs can be associated with multiple queries, 
+all of which will be executed sequentially every time the created job is triggered according to the assigned schedule.
+
+
```

Comment:
We shall add `connection.start_jobs()` and `connection.stop_jobs()` in the documentation.